components/next_steps_view/steps/team_profile_step/team_profile_step.tsx

```diff
@@ -0,0 +1,218 @@
+            <div className='NextStepsView__stepWrapper'>
+                <div className='TeamProfileStep'>
+                    <div className='TeamProfileStep__profilePicture'>
+                        <h3>
+                            <FormattedMessage
+                                id='next_steps_view.team_profile_step.teamImage'
+                                defaultMessage='Team image'
+                            />
+                        </h3>
+                        <PictureSelector
+                            onSelect={this.onSelectPicture}
+                            onRemove={this.onRemovePicture}
+                            src={pictureSrc}
+                            placeholder={(
+                                <div
+                                    data-testid='teamIconInitial'
+                                    className={'TeamIcon__initials TeamIcon__initials__lg'}
+                                    aria-label={'Team Initials'}
+                                >
+                                    {this.props.team.display_name ? this.props.team.display_name.replace(/\s/g, '').substring(0, 2) : '??'}
+                                </div>
+                            )}
+                        />
+                    </div>
+                    <div className='TeamProfileStep__textInputs'>
+                        <h3>
+                            <FormattedMessage
+                                id='next_steps_view.team_profile_step.enterYourNameAndURL'
+                                defaultMessage='Enter your team name'
+                            />
+                        </h3>
+                        <Input
+                            name='teamName'
+                            type='text'
+                            value={this.state.teamName}
+                            onChange={this.handleNameInputChange}
+                            placeholder={Utils.localizeMessage('next_steps_view.team_profile_step.teamName', 'Team name')}
+                            error={this.state.teamNameError}
+                            info={Utils.localizeMessage('next_steps_view.team_profile_step.nameWillBeDisplayed', 'Your team name will be displayed in your sidebar')}
+                        />
+                        {/* TODO: For URL
+                            <Input
+                            name='teamURL'
+                            type='text'
+                            value={this.state.teamURL}
+                            onChange={this.handleURLInputChange}
+                            placeholder={Utils.localizeMessage('next_steps_view.team_profile_step.teamURL', 'Team URL')}
+                            error={this.state.teamURLError}
+                            info={Utils.localizeMessage('next_steps_view.team_profile_step.youCanCustomize', 'You can customize your team url')}
+                            textPrefix={this.getSanitizedSiteURL()}
+                        /> */}
```

Comment:
Should be removed right?

Reply:
I guess for now yeah I'll remove it, wasn't sure if we were gonna bring it back once we have a plan for actually doing this (if it's even possible).